### ISSUE TITLE
Memory leak in itkParticleImplicitSurfaceDomain

### ISSUE THREAD

**Issue by medakk** [SCIInstitute/ShapeWorks#947]
https://github.com/SCIInstitute/ShapeWorks/blob/master/Libs/Optimize/ParticleSystem/itkParticleImplicitSurfaceDomain.txx#L30

meshFIM* is seemingly never freed

**Comment by sheryjoe** [comment]
@medakk @akenmorris is this still an issue?

**Comment by akenmorris** [comment]
Yes, it looks like it is.  However, I do not know how to run this section of code in order to test any changes.  This is for Mesh Based Attributes with PLY/Trimesh files.  Do we have sample data for this?

I don't think anyone is using this, nor does it appear to be a leak that could ever cause a problem.  It's simply that the Trimesh Mesh won't be free'd at the end of the program when all the domains de-allocate.  I guess if you did multiple runs with Studio, but there is no way to specify these extra meshes for mesh based attributes anyway.

I'd recommend pushing this off to a later milestone.

**Comment by sheryjoe** [comment]
We need to come up with feature-based SSM use case, e.g. femurs + cortical thickness

**Comment by akenmorris** [comment]
No longer applicable, there's no meshfim anymore
